pootle/apps/pootle_language/models.py

```diff
@@ -173,14 +173,12 @@ def get_stats_for_user(self, user):
 
         return self.get_stats()
 
-    def get_children_for_user(self, user):
-        translation_projects = self.translationproject_set \
-                                   .for_user(user) \
-                                   .order_by('project__fullname')
-        user_tps = filter(lambda x: x.is_accessible_by(user),
-                          translation_projects)
-
-        return user_tps
+    def get_children_for_user(self, user, select_related=None):
+        tps = self.translationproject_set.for_user(
+            user, select_related=select_related)
+        return filter(
```

Comment:
![Codacy](https://www.codacy.com/versioned/images/favicon.png) Issue found: [map/filter on lambda could be replaced by comprehension](https://www.codacy.com/app/dwayne/pootle/file/1722118520/issues/source?bid=1990595&fileBranchId=3032697#l179)